kategory-core/src/main/kotlin/kategory/instances/NumberInstances.kt

```diff
@@ -26,6 +26,14 @@ object ByteOrderInstanceImplicits {
     @JvmStatic fun instance(): Order<Byte> = ByteOrderInstance
 }
 
+object ByteEqInstance: Eq<Byte> {
+    override fun eqv(a: Byte, b: Byte): Boolean = a == b
+}
+
+object ByteEqInstanceImplicits {
```

Comment:
Can it be generated using @instance()?

Reply:
I did some investigation and this is not possible due to APT limitations. If you try to get the type from the annotation it becomes a java primitive byte and so we don't now the correct type anymore.

https://area-51.blog/2009/02/13/getting-class-values-from-annotations-in-an-annotationprocessor/
```
element.getAnnotation(instanceAnnotationClass).let {
    try {
        elementUtils.getTypeElement(it.target.qualifiedName)
    } catch (mte: MirroredTypeException) {
        mte.typeMirror //byte
    }
}
```